gen/dcos-config.yaml

```diff
@@ -836,6 +841,11 @@ package:
                   "Port": 5051,
                   "Uri": "/overlay-agent/overlay",
                   "Role": ["agent", "agent_public"]
+              },
+              {
+                  "Port": 5051,
+                  "Uri": "/containers",
+                  "Role": ["agent", "agent_public"]
               }
```

Comment:
What info do we get from `containers` that we didn't have in `state`?

Reply:
Good question @philipnrmn . I'm not sure about @janisz's motivation, but I have used containers endpoint to get executor_pid in the past which I needed to kill a stuck task.

Reply:
It was requested by supporting team. I'm not sure why.